src/resolver/message.ts

```diff
@@ -41,20 +41,21 @@ export class Message {
     let message = '';
 
     // tags
-    for (let i = 0; i < this.tags.length; i++) {
+    for (let i = 0; i < Message.tags.length; i++) {
       if (i > 0) {
         message += ' ';
       }
-      message += '#' + this.tags[i];
+      message += '#' + Message.tags[i];
     }
 
-    message += '\n';
+    message += '\n\n';
 
     // info
-    message += 'Title: ' + this.title + '\n';
-    message += 'Author: ' + this.author + '\n';
-    message += 'Publisher: ' + this.publisher + '\n';
-    message += this.url;
+    // Bookcrush format
+    message += 'Title: <code>' + Message.title + '</code>' + '\n';
+    message += 'Author: <code>' + this.author + '</code>' + '\n';
+    message += 'Publisher: <i>' + this.publisher + '</i>' + '\n\n';
+    message += '<a href = "' + this.url + '">Link</a>';
```

Comment:
Is this format OK with the other admins?

Reply:
yes. basically i retained the same format but just added the formatting (mono and hyperlink)